components/mdc/Checkbox/Checkbox.svelte

```diff
@@ -19,26 +19,48 @@ export let disabled = false
 export let uppercase = false
 /** @type {string} random id prefixed with checkbox- for the input id*/
 export let inputID = generateRandomID('checkbox-')
+/** @type {string} name attribute for the checkbox input - important for forms */
+export let name = ''
+/** @type {string} value attribute when checked (default is "on") */
+export let value = 'on'
 
 let checkboxElement = {}
 let formFieldElement = {}
 let checkbox
 
-$: if (checkbox) checkbox.checked = checked
-
 onMount(() => {
   checkbox = new MDCCheckbox(checkboxElement)
   const formField = new MDCFormField(formFieldElement)
-
   formField.input = checkbox
+
+  checkbox.checked = checked
 })
 
-const handleChange = () => dispatch(checkbox.checked ? 'checked' : 'unchecked')
+$: if (checkbox && checkbox.checked !== checked) {
```

Comment:
Can this be `checkbox?.checked`?

Reply:
yup. Thanks

Reply:
actually I don't think that's equivalent because if checkbox is undefined then the block won't run, but if checked is undefined it will as long as checked isn't undefined.